### ISSUE TITLE
Current value for lazy-loaded elements will be not set

### ISSUE THREAD

**Issue by dobexx** [trofrigo/global-variable#3]
Hello,

later loaded elements that use this element, get not the current value at run time. Only if one element set the value later. Is the intended?

For example:

1. Element with first global-variable set the value to true
2. Lazy loaded element that use the same key becomes only undefined as value.

**Comment by rodrigosancho** [comment]
Could you show me some code please? 

Thank you!

**Comment by dobexx** [comment]
Ok no problem:

Main element:

```
<dom-module id="xi-selfcare"> 
	
	<template strip-whitespace>		
		<global-variable key="api_base" value="{{_apiBase}}"></global-variable>
		<xi-second></xi-second>
	</template>
	
	<script>
		Polymer({
			is: "xi-selfcare",
			properties: {
				_apiBase: { type: String },
			},
			ready: function() {
				this._apiBase = '/api/v1';

        		   	Polymer.Base.importHref('xi-second.html', function() {
        	    			...
        			});
			}
		});
	</script>

</dom-module>
```

Second element:

```
<dom-module id="xi-second"> 
	
	<template strip-whitespace>		
		<global-variable key="api_base" value="{{_apiBase}}" readonly></global-variable>
	</template>
	
	<script>
		Polymer({
			is: "xi-second",
			properties: {
				_apiBase: { type: String },
			},
			ready: function() {
				console.log(this._apiBase); // variable is now undefined and later too!
			}
		});
	</script>

</dom-module>

```

**Comment by rodrigosancho** [comment]
My fault!!

I checked the code and realized that I was not initializing the `value` in the global-variable's subscribe private method. I fixed it in the [v0.1.3](https://github.com/trofrigo/global-variable/releases/tag/v0.1.3) release.

Sorry about that.

**Comment by dobexx** [comment]
Thank you for the fast fix.
